File: src/main/java/ru/nspk/sbp/trm/config/TestResourceManagerConfigurationProperties.java
package ru.nspk.sbp.trm.config;

import lombok.Data;
import lombok.extern.slf4j.Slf4j;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.context.properties.ConfigurationProperties;

import javax.annotation.PostConstruct;
x@Data
@Slf4j
@ConfigurationProperties(prefix = "test.resource.manager.bot")
public class TestResourceManagerConfigurationProperties {
    @Value("${path}")
    private String botPath;
    @Value("${username}")
    private String botUsername;
    @Value("${token}")
    private String botToken;

    @PostConstruct
    void printConfig() {
        log.info("Using properties: " + this);
    }

}

File: src/main/kotlin/trm/config/TestResourceManagerConfigurationProperties.kt
package trm.config

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "test.resource.manager.bot")
data class TestResourceManagerConfigurationProperties(
    var path: String = "",
    var username: String = "",
    var token: String = ""
)